### ISSUE TITLE
 in the last nightly DSC port doesn't work

### ISSUE THREAD

**Issue by tonims1** [EdgeTX/edgetx#1138]
In the lastest nightlies It have no ppm output on the top DSC port, 
Simulator does not work for me

**General information**
 - EdgeTX version (latest nightlies )
 - Hardware (RMTX16S)

**Comment by raphaelcoeffic** [comment]
@tonims1 the TX16S has quite some issues with the physical connection. You might want to have a look at that. @rotorman has described some of the issues on RCG.

**Comment by pfeerick** [comment]
Actually, looks like something may have happened... DSC simulator cable is working with 2.5.0, but not a nightly from probably two days ago. er... the trainerData block don't look right... 

```yml
trainerData: 
   mode: SLAVE
   channelsStart: 0
   channelsCount: 0
   frameLength: 0
   delay: 0
   pulsePol: 0
```
![image](https://user-images.githubusercontent.com/5500713/143190359-d35e36a8-f596-4699-af46-95eeb7a25541.png)

**Comment by raphaelcoeffic** [comment]
Ouch!

**Comment by pfeerick** [comment]
Well that makes sense, given when I moved everything one value off they changed +1 or -1 ... but DSC not working does appear to be firmware related ;)

**Comment by rotorman** [comment]
Problem confirmed with RM TX16S (analog sticks) and today's nightly. With Trainer set to Slave/Jack this is what I see on logic analyzer (does NOT resemble PPM):

![grafik](https://user-images.githubusercontent.com/21011587/143617052-a081450b-5e6b-4f6f-802d-99454e601f5c.png)

(bottom line is the same signal, just sampled with analog circuitry).

**Comment by raphaelcoeffic** [comment]
Fixed by 453e478
